front-end/src/app/register/register.component.html

```diff
@@ -28,7 +28,7 @@
           <input type="text" class="form-control" id="zip" placeholder="Zip Code" required/>
         </div>
         <div class="form-group">
-          <button type="submit" class="btn btn-primary">Submit</button>
+          <button type="submit" class="btn btn-primary" (click)="goToProfile()">Submit</button>
```

Comment:
form tag should be #registerForm="ngForm" with an onSubmit function. I think Tycho will address this soon.